src/modules/site-v2/templates/strain/landing_page.html

```diff
@@ -2,12 +2,60 @@
 
 {% block content %}
 
+<style>
+    h2, h3 {
```

Comment:
This custom CSS is creating some conflicts with the footer text so should be removed. I'll add some comments below on how to achieve the same effects using Bootstrap classes.